.gitignore

```diff
@@ -3,3 +3,4 @@
 /tests/integrations/*/node_modules
 /node_modules
 /test.*
+yarn.lock
```

Comment:
I wouldn't ignore yarn.lock, we might want to add it to repository

Reply:
I thought it'd be safer to add it to `.gitignore` until an explicit decision to include it had been made, at which time it could be easily removed. Do you disagree?